server/lib/routes/get-metrics-errors.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var logger = require('mozlog')('server.metrics-errors');
+var request = require('request');
+
+var config = require('../configuration');
+var sentryConfig = config.get('sentry');
+
+/**
+ * This module serves as a proxy for front-end errors.
+ * The errors are sent to Sentry service for triage.
+ *
+ * The content server sends a GET request via forked version of raven-js, i.e GET /metrics-errors?sentry_version=4&...
+ * The query parameters are forwarded to the appropriate Sentry metrics dashboard.
+ *
+ * @returns {{method: string, path: string, process: Function}}
+ */
+module.exports = function () {
+
+  return {
+    method: 'get',
+    path: '/metrics-errors',
+    process: function (req, res) {
+      // respond right away
+      res.json({ success: true });
+
+      var indexOfQuery = req.url.indexOf('?');
+
+      if (sentryConfig && sentryConfig.enabled && indexOfQuery >= 0) {
+
+        // TODO: we can probably include `release` version of the content server
+        // see https://raven-js.readthedocs.org/en/latest/config/index.html#release
+
+        // send the proxied request to sentry
+        var query = req.url.substr(indexOfQuery + 1);
+        // set API_KEY using the server
+        query = query.replace('__API_KEY__', sentryConfig.api_key);
```

Comment:
> So the trick here is not to expose the api key in the client code :)

This is precisely what I was thinking in https://github.com/mozilla/fxa-content-server/pull/2529/files#r31878271